frontend/src/components/vuetify/Atoms/ListElement.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <v-card class="mx-auto" width="400">
+    <v-list lines="two">
+      <v-list-item
+        v-for="(item, index) in items"
+        :key="index"
```

Comment:
```suggestion
        :key="item.MeetingID"
```
Using index as key can lead to problems when the list is updated.